### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by sackh**
Signed-off-by: Kharude, Sachin <sachin.kharude@here.com>

### CODE REVIEW

File: docs/source/contributing.rst [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+Seven Steps for Contributing
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are seven basic steps to contributing to *xyzspaces*:
+
+1) Fork the *xyzspaces* git repository
+2) Create a development environment
+3) Install *xyzspaces* dependencies
+4) Make changes to code and add tests
+5) Run liniting
```
Comment (deeplook) [inline docs/source/contributing.rst:48]: Typo?

File: docs/source/contributing.rst [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+
+Running the test suite
+~~~~~~~~~~~~~~~~~~~~~~
+
+The tests can then be run directly inside your Git clone (without having to
+install *xyzspaces*) by typing::
+
+    pytest -v --cov=xyzspaces tests
+
+5) Run liniting
```
Comment (deeplook) [inline docs/source/contributing.rst:219]: Typo?

**Review by deeplook — COMMENTED** [review]
See some typos.

File: docs/source/contributing.rst (hunk 2 above)
Comment (sackh) [reply docs/source/contributing.rst:219]: Do you mean linting? I think it is correct.

Comment (deeplook) [reply docs/source/contributing.rst:219]: I guess it should be linting...

File: docs/source/contributing.rst (hunk 1 above)
Comment (sackh) [reply docs/source/contributing.rst:48]: https://code.visualstudio.com/docs/python/linting#:~:text=Linting%20highlights%20syntactical%20and%20stylistic,that%20can%20lead%20to%20errors.

File: docs/source/contributing.rst (hunk 2 above)
Comment (sackh) [reply docs/source/contributing.rst]: sorry, my bad. fixed typo.

File: docs/source/contributing.rst (hunk 1 above)
Comment (sackh) [reply docs/source/contributing.rst]: fixed typo.

**Review by deeplook — APPROVED** [review]

**Pull request merged**